java/src/main/java/com/logicalclocks/hsfs/FeatureView.java

```diff
@@ -99,8 +102,19 @@ public FeatureViewBuilder description(String description) {
       return this;
     }
 
+    /**
+     * Query of a feature view. Note that `as_of` argument in the `Query` will be ignored because feature view does
+     * not support time travel query.
+     *
+     * @param query
+     * @return builder
+     */
     public FeatureViewBuilder query(Query query) {
       this.query = query;
+      if (query.isTimeTravel()) {
+        LOGGER.info("`as_of` argument in the `Query` will be ignored because" +
+            " feature view does not support time travel query.");
```

Comment:
Stylecheck still wants to thave the `+` in the next line:
`C:\Users\doehmen\Documents\Hopsworks\feature-store-api\java\src\main\java\com\logicalclocks\hsfs\FeatureView.java:115:79: '+' should be on a new line. [OperatorWrap]`